cmd/internal/cli/inspect_linux.go

```diff
@@ -178,6 +186,11 @@ var InspectCmd = &cobra.Command{
 
 		a := []string{"/bin/sh", "-c", ""}
 
+		if apps {
+			sylog.Debugf("Inspecting apps in container")
```

Comment:
Can we change this to `Inspection of apps selected.` So that all the debug statements are really similar?